R/render_executive_summary.R

```diff
@@ -6,6 +6,7 @@
 #' @param exec_summary_dir Character single, valid filepath to a directory that contains the template, e.g. `system.file("extdata", "PA2022CH_en_exec_summary", package = "pacta.executive.summary")`
 #' @param survey_dir Character single, valid filepath to a directory that contains the survey files for the user
 #' @param score_card_dir Character single, valid filepath to a directory that contains score card files for the user
+#' @param analysis_inputs_dir: Character single, valid filepath to a PACTA analysis results directory that contains the SBTi data merged with financial data in the COP case
```

Comment:
```suggestion
#' @param analysis_inputs_dir Character single, valid filepath to a PACTA analysis results directory that contains the SBTi data merged with financial data in the COP case
```

colon makes the argument name unrecognizable

Reply:
Thanks for catching that. I don't even know how I came up with this implementation. I'm tired.